commit a72cbf3c2395edfe0009ba232fee7a1f8e71a91a
Author: q66 <daniel@octaforge.org>
Date:   Fri Jun 21 03:34:27 2019 +0200

    lib/package_unpack.c: silence gcc false positive
    
    On some systems, something like this happens:
    
    package_unpack.c:375:11: error: 'buf' may be used uninitialized in this function
    
    This is not actually a bug as logically `buf` is always initialized
    in that place, but gcc doesn't like it anyway.

diff --git a/lib/package_unpack.c b/lib/package_unpack.c
index ecebc694..e995d40c 100644
--- lib/package_unpack.c
+++ lib/package_unpack.c
@@ -84,7 +84,7 @@ unpack_archive(struct xbps_handle *xhp,
 	size_t  instbufsiz = 0, rembufsiz = 0;
 	ssize_t entry_size;
 	const char *entry_pname, *transact, *binpkg_pkgver;
-	char *pkgname, *buf;
+	char *pkgname, *buf = NULL;
 	int ar_rv, rv, error, entry_type, flags;
 	bool preserve, update, file_exists, keep_conf_file;
 	bool skip_extract, force, xucd_stats;
